content/analyst/inbox-and-design-tracker/how-to-use-analytics-tracker.md

```diff
@@ -1,63 +1,56 @@
 
-![](media/how_to_use_analytics_tracker/image_1.png)
+- Enter a tracking code name
+- Set an expiration date for tracking this campaign
+- Select your Marketing Software to proceed to the next step.
 
- Next, select your email deployment software (If you don’t see your ESP in the list, select “Other” and input the merge tag that your provider uses to dynamically publish the recipient’s email address into your email content. Example: %%emailaddr%%)
+![](media/how_to_use_analytics_tracker/trackingcode.png)
 
-![](media/how_to_use_analytics_tracker/image_2.png)
+**Step 2. Place Tracking Code In Your Email**
 
- Copy the tracking code and place it into your HTML. Once added, send a preview of your email campaign to the test address shown (Pro Tip: If this isn’t your first rodeo, “Skip this” to end the tracking code setup):
+- From the pulldown, select your email deployment software (If you don’t see your ESP in the list, select “Other” and input the merge tag that your provider uses to dynamically publish the recipient’s email address into your email content. Example: %%emailaddr%%)
+- Copy the tracking code and place it into your HTML and select Done. (If you want to verify the tracking code before deployment select the "Verify Code" button.)
+- After you email the campaign, come back to Analytics Tracker and view the results for the campaign.
 
-![](media/how_to_use_analytics_tracker/image_3.png)
+![](media/how_to_use_analytics_tracker/trackingcode2.png)
 
- Once you’ve mailed the campaign, come back to Analytics Tracker and view the results for the campaign. Each time one of your subscriber's opens the email containing the pixel, that is one Analytics event. Data will continue to be collected until the expiration date that you defined in the tracking code setup.
+*Note: Each time one of your subscriber's opens the email containing the pixel, that is one Analytics event. Data will continue to be collected until the expiration date that you defined in the tracking code setup.*
 
 ## Scheduled Analytics Exports
 
  We've made exporting Analytics campaigns a whole lot easier! You can now schedule an events export for 'daily' or 'weekly' during an active campaign, or 'upon expiration' of the campaign tracking.
 
  Here is a look at the all the export options that are available:
 
-![](media/how_to_use_analytics_tracker/image_4.png)
+![](media/how_to_use_analytics_tracker/export.png)
 
  Use your findings to prioritize optimization of your email template across email clients, browser and devices. Understand where your subscribers are located throughout the world and how much time they spend viewing your email.
 
-![](media/how_to_use_analytics_tracker/image_5.png)
+![](media/how_to_use_analytics_tracker/counts.png)
 
 
-![](media/how_to_use_analytics_tracker/image_6.png)
+![](media/how_to_use_analytics_tracker/engagement.png)
 
 
-![](media/how_to_use_analytics_tracker/image_7.png)
+![](media/how_to_use_analytics_tracker/EmailClients.png)
```

Comment:
I _think_ the standard for this repo is lower case filenames, not sure if that is because of how netlify does things, the build, or just a standard to make everything similar

Reply:
updated to emailclients.png